AirCasting/RootAppView.swift

```diff
@@ -17,6 +17,7 @@ struct RootAppView: View {
     @StateObject private var bluetoothManager = BluetoothManager(mobilePeripheralSessionManager: MobilePeripheralSessionManager(measurementStreamStorage: CoreDataMeasurementStreamStorage(persistenceController: PersistenceController.shared)))
     
     @StateObject private var userSettings = UserSettings()
+    @StateObject var selectedSection = SelectSection()
```

Comment:
Is there a reason for declaring it here instead of MainTabBarView?

Reply:
In some point of my mind process it was, but I will check if it is still the case 😄